docs/user-installation-review.md

```diff
@@ -0,0 +1,180 @@
+# OpenShift Provider Certification Tool - Installation Review
+
+> WIP - this document is working in progress
+
+> TODO: steps describing in detail the topics to review before submitting the results.
+
+## Compute
+
+> TODO: any compute specific?
+
+- Minimal required for Compute nodes: [User Documentation -> Pre-requisites](./user.md#prerequisites)
+
+## Load Balancers
+
+- Private Load Balancers
+
+The basic OpenShift Installations with support of external Load Balancers are used to deploy 3 Load Balancers: public and private for control plane services (Kubernetes API and Machine Config Server), and one public for the ingress.
+
+The address for the private Load Balancer must point to the DNS `api-int.<cluster>.<domain>`, which will be accessed for internal services.
+
+Reference: [User-provisioned DNS requirements](https://docs.openshift.com/container-platform/4.11/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-dns-user-infra_installing-platform-agnostic)
+
+### Health Checks
+
+The kube-apiserver has a graceful termination engine that requires the Load Balancer health check probe to HTTP path.
+
+| Service | Protocol | Port | Path | Threshold | Interval | Timeout |
+| -- | -- | -- | -- | -- | -- | -- |
+| Kubernetes API Server | HTTPS* | 6443 | /readyz | 2  | 10 | 10 |
+| Machine Config Server | HTTPS* | 22623 | /healthz | 2  | 10 | 10 |
+| Ingress | TCP | 80 | - | 2  | 10 | 10 |
+| Ingress | TCP | 443 | - | 2  | 10 | 10 |
+
+> Note/Question: Not sure if we need to keep the HTTP (non-SSL on the doc). In the past, I talked with KAS team and he had plans to remove that option, but due to the limitation of a few cloud providers it will not. Some providers that still use this: [Alibaba](https://github.com/openshift/installer/blob/master/data/data/alibabacloud/cluster/vpc/slb.tf#L31), [GCP Public](https://github.com/openshift/installer/blob/master/data/data/gcp/cluster/network/lb-public.tf#L20-L21)
+*It's required to health check support HTTP protocol. If the Load Balancer used does not support SSL, alternatively and not preferably you can use HTTP - but never TCP:
+
+| Service | Protocol | Port | Path | Threshold | Interval | Timeout |
+| -- | -- | -- | -- | -- | -- | -- |
+| Kubernetes API Server | HTTP* | 6080 | /readyz | 2  | 10 | 10 |
+| Machine Config Server | HTTP* | 22624 | /healthz | 2  | 10 | 10 |
+
+
+- Flavor/Size
+
+> TODO: Need to check if we have any information in our documentation. I am considering the current deployments of Alibaba and NLB (auto-scaling)
```

Comment:
We have platform agnostic documentation on LB here. Are you looking for something specifically about throughput? 

https://docs.openshift.com/container-platform/4.11/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-load-balancing-user-infra_installing-platform-agnostic

Reply:
Yeah, I didn't find anything related to "minimum throughput required for the Load Balancer" in the official documentation. It's something interesting as the default used on our review was lower than the utilization in reference cloud (AWS internal NLB), for that reason I created that section.